docs/quickstart/csharp.md

```diff
@@ -215,18 +217,19 @@ Note that you may also have to change the permissions of the protoc and protobuf
 binaries in the `Grpc.Tools` package under `examples/csharp/helloworld/packages`
 to executable in order to run the commands below.
 
-From the `examples/csharp/helloworld` directory:
+From the `examples/csharp/helloworld` directory, or the `examples/csharp/helloworld-from-cli` directory if using the .NET Core SDK:
 
 **Windows**
 
 ```
-> packages/Grpc.Tools.1.0.0/tools/windows_x86/protoc -I../../protos --csharp_out Greeter --grpc_out Greeter ../../protos/helloworld.proto --plugin=protoc-gen-grpc=packages/Grpc.Tools.1.0.0/tools/windows_x86/grpc_csharp_plugin.exe
+> ./generate_protos.bat
```

Comment:
When trying to simplify the scripts to use shell variables, I found it a lot simpler to use the bat than a powershell command. (easier to sub in GRPC_TOOLS package and version into the command as a variable in the bat script).

But I see that I had the wrong idea in changing these commands, so I'm not sure if its still useful.